### DISCUSSION TITLE
zk-nft-exchange

### DISCUSSION THREAD

**Discussion by baidang201** [zkp-co-learning/zkp-co-learn#58] [I've got an idea!]
一句话 IDEA： 基于Polygon zkEVM的nft互换交易所

隐藏交易对的提交人
隐藏交易对的nft信息

技术栈
（circom snarkjs solidity typejs metamask）

我的角色：技术设计，项目/产品经理，项目部署
人数需求（不包含发起者）：共需3人，1人 web3前端经验  2人zk及solidity开发者
我的联系方式：tg smithwow
个人简介：5年区块链开发，有rust/nodejs经验，zp刚入门

现阶段3-4人的团队较为合适, 期待加入, 一起交流和学习zkp

参考：
https://vivianblog.hashnode.dev/how-to-create-a-zero-knowledge-dapp-from-zero-to-production

**Comment by luckyyang** [comment]
好奇交易对的提交人和nft信息如何隐藏啊？

需求上，如果隐藏购买人的信息是不是更需要？

**Comment by baidang201** [comment]
现在还不知道如何做到，调研中

隐藏提交人和提交的nft信息，类似中心化交易所的暗盘，提交交易后不增加深度，防止被狙击
